src/dodal/devices/areadetector/adaravis_deadtime.csv

```diff
@@ -0,0 +1,36 @@
+model,deadtime
+# additional fields passed as a list
+# cite: https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/Manta/techman/Manta_TechMan.pdf retrieved 2024-04-05
+Manta G-125,63e-6
+Manta G-145,106e-6
+
+Manta G-235,118e-6,Mono8,Mono12Packed,BayerRG8,BayerRG12,BayerRG12Packed,YUV411Packed
+Manta G-235,256e-6,Mono12,BayerRG12,YUV422Packed
+Manta G-235,390e-6,RGB8Packed,BGR8Packed,YUV444Packed
+
+Manta G-895,404e-6,Mono8,Mono12Packed, BayerRG8,BayerRG12Packed,YUV411Packed
+Manta G-895,542e-6,Mono12,BayerRG12,YUV422Packed
+Manta G-895,822e-6,RGB8Packed,BGR8Packed,YUV444Packed
+
+Manta G-2460,979e-6,Mono8,Mono12Packed,BayerRG8,BayerRG12Packed,YUV411Packed
+Manta G-2460,1304-e6,Mono12,BayerRG12,YUV422Packed
+Manta G-2460,1961e-6,RGB8Packed,BGR8Packed,YUV444Packed
+
+# cite: https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/various/appnote/GigE/GigE-Cameras_AppNote_PIV-Min-Time-Between-Exposures.pdf retrieved 2024-04-05
+Manta G-609,47e-6
+
+#cite: https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/Mako/techman/Mako_TechMan_en.pdf retrieved 2024-04-05
+Mako G-040,101e-6,Mono8,Mono12Packed,BayerRG8,BayerRG12Packed,YUV411Packed
+Mako G-040,140-e6,Mono12,BayerRG12,YUV422Packed
+Mako G-040,217-6,RGB8Packed,BGR8Packed,YUV444Packed
+
+Mako G-125,70e-6
+
+# Assume 12 bits: 10 bits = 275e-6
+Mako G-234,356-6,Mono8,BayerRG8,BayerRG12,BayerRG12Packed,YUV411Packed,YUV422Packed
+# Assume 12 bits: 10 bits = 563e-6
+Mako G-234,726-e6,RGB8Packed,BRG8Packed,YUV444Packed
```

Comment:
I'm surprised this file parsed properly...

```suggestion
Manta G-2460,1304e-6,Mono12,BayerRG12,YUV422Packed
Manta G-2460,1961e-6,RGB8Packed,BGR8Packed,YUV444Packed

# cite: https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/various/appnote/GigE/GigE-Cameras_AppNote_PIV-Min-Time-Between-Exposures.pdf retrieved 2024-04-05
Manta G-609,47e-6

#cite: https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/Mako/techman/Mako_TechMan_en.pdf retrieved 2024-04-05
Mako G-040,101e-6,Mono8,Mono12Packed,BayerRG8,BayerRG12Packed,YUV411Packed
Mako G-040,140e-6,Mono12,BayerRG12,YUV422Packed
Mako G-040,217e-6,RGB8Packed,BGR8Packed,YUV444Packed

Mako G-125,70e-6

# Assume 12 bits: 10 bits = 275e-6
Mako G-234,356e-6,Mono8,BayerRG8,BayerRG12,BayerRG12Packed,YUV411Packed,YUV422Packed
# Assume 12 bits: 10 bits = 563e-6
Mako G-234,726e-6,RGB8Packed,BRG8Packed,YUV444Packed
```

Reply:
It's just a csv until a value is actually selected, good catches, there's a reason I'm not in data entry